### PR TITLE
FYLE-8zd53k: Reports list view changes - 1

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

**Comment by Dimple16** [comment]
Screenshot 2021-09-07 at 12 51 58 AM

File: src/app/fyle/my-reports/my-reports-card/my-reports-card.component.scss [hunk 1]
```diff
@@ -21,62 +32,60 @@
...
-    display: inline-block;
     color: white;
-    border-radius: 4px;
-    padding: 2px 8px;
-    text-transform: uppercase;
+    border-radius: 16px;
+    padding: 2px 16px;
     font-size: 12px;
-    font-weight: 500;
+    text-transform: capitalize;
```
Comment (Julias0) [inline src/app/fyle/my-reports/my-reports-card/my-reports-card.component.scss:88]: we have a text-capitalize class, you can use that

File: src/app/fyle/team-reports/team-reports.page.html [hunk 2]
```diff
@@ -93,9 +93,11 @@
...
     <div class="reports--zero-state">
       <app-fy-zero-state
-        [image]="'../../../assets/images/zero-states/coffee.svg'"
-        [header]="'Nothing here, grab a coffee maybe?'"
-        [message]="'Expense reports that need your approval will show up here.'"
+        [unscaledImage]="true"
+        [useNewStyling]="true"
+        image="../../../assets/images/zero-states/expenses.png"
+        header="Get yourself a coffee &#9749;"
+        message="No reports pending on you for approval."
```
Comment (Julias0) [inline src/app/fyle/team-reports/team-reports.page.html:100]: lets also change the content here - check with Sai on this

**Review by Julias0 — APPROVED** [review]

File: src/app/fyle/team-reports/team-reports.page.html (hunk 2 above)
Comment (Dimple16) [reply src/app/fyle/team-reports/team-reports.page.html:100]: this is the same as the new design, will check with Sai

**Pull request merged**